### ISSUE TITLE
Implement automatic tagging on release

### ISSUE THREAD

**Issue by d11wtq** [d11wtq/dockerpty#40]
Apparently dockerpty is being packaged for Debian and they're using the Git tags to find versions. 0.3.3 wasn't tagged.

I guess this should probably be automatic or something really via a Makefile or equivalent process.

**Comment by aanand** [comment]
Yeah, we can probably make this more automatic. @shin- - how does docker-py do it?

**Comment by shin-** [comment]
I tag things manually as part of the release process. Sorry, no magic here! :)
